tools/xdp/ebpf/rust-toolchain.toml

```diff
@@ -0,0 +1,6 @@
+[toolchain]
+# pin the version to prevent random breakage
+channel = "nightly-2023-02-20"
```

Comment:
Do more recent versions not work, or is this just when you started this?

Reply:
That's when I started it. I can update if you want.